src/client/pages/OfferNew/index.tsx

```diff
@@ -44,24 +45,28 @@ export const OfferNew: React.FC = () => {
   const localeIsoCode = getLocaleIsoCode(currentLocale)
   const currentMarket = useMarket()
   const variation = useVariation()
-  const quoteIds = storage.session.getSession()?.quoteIds ?? []
+  const history = useHistory()
   const { data: redeemedCampaignsData } = useRedeemedCampaignsQuery()
   const redeemedCampaigns = redeemedCampaignsData?.redeemedCampaigns ?? []
+  const quoteIds = useQuoteIds(storage, history)
   const { data, loading: loadingQuoteBundle, refetch } = useQuoteBundleQuery({
     variables: {
       input: {
-        ids: [...quoteIds],
+        ids: [...(quoteIds ?? [])],
       },
       locale: localeIsoCode,
     },
+    skip: quoteIds === null,
   })
 
-  const history = useHistory()
   const checkoutMatch = useRouteMatch(
     '/:locale(se-en|se|no-en|no|dk-en|dk)/new-member/sign',
   )
   const toggleCheckout = createToggleCheckout(history, currentLocale)
 
+  if (quoteIds === null) {
```

Comment:
Can `quoteIds` ever be `null`? Doesn't the `useQuoteIds` hook always return an array? Or am I missing something?

Reply:
It's returning null for the initial state, which I used to distinguish between the states `NOT LOADED` and `LOADED, BUT NO DATA`. I guess it could be made more explicit, however 🤔

Reply:
Ah yes of course! 🙈 But yes, maybe it would be an idea to also return an `isLoading` state from the hook instead? That would be a way to be pretty explicit?

Also, realised that we already do a return of `<LoadingPage>` on line 81 so this condition (whether we leave it as `quoteIds === null`  or do an `isLoading`) should probably just be added to line 80. Personally I prefer to only have one `return` statement per component and do all the conditioning in the JSX but I know we do both ways in this repo so that change is probably unnecessary since we haven't yet agreed on one way or the other in the web team 😄